docs/sct-events.md

```diff
@@ -3,3 +3,18 @@
 ## Overview
 
 ![Overview](./sct_events.png?raw=true "Overview")
+
+### Event Analyzer
```

Comment:
@bentsi 
this is the section for the email, to be sent once we merge this

anything else you thing we should put here ?